### PR TITLE
Describe sampling configuration

### PR DESCRIPTION
**Pull request by anton-ryzhov**
Describe sampling configuration

### CODE REVIEW

**Comment by anton-ryzhov** [comment]
:+1:

**Comment by pheanex** [comment]
:+1:

File: components/taupage.rst [hunk 1]
```diff
@@ -602,12 +602,10 @@ Sample logging configuration::
          authlog_destination: scalyr
          scalyr_region: eu
          scalyr_account_key: "aws:kms:XYZABC..."
+         scalyr_agent_applog_sampling: '[{ match_expression: "INFO", sampling_rate: 0.1 }, { match_expression: "FINE", sampling_rate: 0 }]'
 
 In this example everything but the ``auth.log`` is logged to ``s3``, the ``auth.log`` is logged to ``Scalyr``. Logging is done with ``Fluentd``.
 
-.. NOTE::
-   If a logging section is present in ``senza.yaml`` then ``scalyr_account_key``, ``scalyr_application_log_parser``, ``scalyr_custom_log_parser`` and ``scalyr_region`` entries outside the scope of the logging section will be ignored.
-
```
Comment (christianberg) [inline components/taupage.rst:610]: To refresh my memory: If one of these is set on the top level, but not in the `logging` section, the top level value will be taken. If they are set both on the top level and in the `logging` section, will value in the logging section will take precedence. Correct?

Comment (anton-ryzhov) [reply components/taupage.rst:610]: Correct
https://github.com/zalando-stups/taupage/pull/565/files#diff-246f63653c4d8d9795a89e0b2ac9f5e3R51

**Pull request merged**